panel/pane/media.py

```diff
@@ -19,6 +19,25 @@
 from .base import ModelPane
 
 
+class TensorLikeMeta(type):
+    """See https://blog.finxter.com/python-__instancecheck__-magic-method/"""
+    def __instancecheck__(self, instance):
+        numpy_attr = getattr(instance, "numpy", "")
+        dim_attr = getattr(instance, "dim", "")
+        return (
+            bool(numpy_attr) and
+            callable(numpy_attr) and
+            callable(dim_attr) and
+            hasattr(instance, "dtype")
+        )
+
+# When support for Python 3.7 is dropped, Torchlike should be implemented as a typing.Protocol
```

Comment:
```suggestion
# When support for Python 3.7 is dropped, Torchlike should be implemented as a typing.Protocol
```

Panel 1.0 does not support 3.7

Reply:
Honestly don't know enough about it to say. I'll merge because I'm about to tag 1.0 but please follow up on that.